datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -1243,6 +1257,13 @@ private void performPartitionMovement(Long notifyTimestamp) {
     return newAssignmentsByInstance;
   }
 
+  void performTaskPostBecomingLeader() {
+    _log.info("performTaskPostBecomingLeader called");
+    int orphanCount = _adapter.cleanUpOrphanConnectorTasks(_config.getZkCleanUpOrphanConnectorTask());
+    _dynamicMetricsManager.createOrUpdateMeter(MODULE, "performTaskPostBecomingLeader",
```

Comment:
If you plan to rename "performTaskPostBecomingLeader", can you also add a static constant for this string literal?

Reply:
Please correct me if I understand something wrong. This pattern is followed in the entire file for  createOrUpdateMeter.

Reply:
Just thought it'll be cleaner to have this as a const separately, but sure if you've done it this way for consistency it's your call. I would generally prefer making all string literals constants, but that is just me.